tests/test_zt2rw.py

```
@@ -0,0 +1,402 @@
+
+    @patch("zotero2readwise.zt2rw.get_zotero_client")
+    @patch("zotero2readwise.zt2rw.Readwise")
+    @patch("zotero2readwise.zt2rw.ZoteroAnnotationsNotes")
+    def test_initialization_with_filters(
+        self, mock_zan_class, mock_rw_class, mock_get_client, zotero_credentials, readwise_token
+    ):
+        """Test initialization with color and tag filters."""
+        mock_client = Mock()
+        mock_get_client.return_value = mock_client
+
+        zt_rw = Zotero2Readwise(
```

Ruff (0.14.3)

Local variable `zt_rw` is assigned to but never used

Remove assignment to unused variable `zt_rw`

(F841)